src/close-test.js

```diff
@@ -14,13 +14,11 @@ const multiaddr = require('multiaddr')
 
 const mh = multiaddr('/ip4/127.0.0.1/tcp/10000')
 
-function closeAndWait (stream) {
+function closeAndWait (stream, callback) {
   pull(
     pull.empty(),
     stream,
-    pull.onEnd((err) => {
-      expect(err).to.not.exist.mark()
-    })
+    pull.onEnd(() => callback())
```

Comment:
This change is not correct, it means it ignores any errors produced by the stream.